app/views/sheet.html

```diff
@@ -1,3 +1,6 @@
+<section class="logout-container">
+	<a href="/logout" role="button" aria-label="Logout">Logout</a>
```

Comment:
Do you think we need it here too, since it's already on `main`?

Reply:
Probably not necessary - my thought was that the sheet was where people were likely to spend a lot of their time, so it would be convenient to have the button there rather than needing to navigate back to main first.  However, there's little difference between that and simply clicking to return home before logging out, so I'd say whatever you think is best.